packages/start/package.json

```diff
@@ -1,84 +1,68 @@
 {
   "name": "@solidjs/start",
-  "version": "1.1.7",
+  "version": "2.0.0-devinxi.0",
   "type": "module",
-  "author": "Ryan Carniato",
-  "license": "MIT",
-  "homepage": "https://start.solidjs.com/",
-  "bugs": "https://github.com/solidjs/solid-start/issues",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/solidjs/solid-start.git",
-    "directory": "packages/start"
-  },
-  "publishConfig": {
-    "access": "public"
-  },
   "scripts": {
-    "typecheck": "tsc --noEmit",
-    "build": "node build && pnpm tsc"
+    "build": "tsc"
   },
-  "files": [
-    "config",
-    "dist",
-    "env.d.ts"
-  ],
-  "types": "./dist/index.d.ts",
   "exports": {
-    ".": "./dist/index.jsx",
-    "./config": "./config/index.js",
-    "./middleware": "./dist/middleware/index.jsx",
-    "./router": "./dist/router/index.jsx",
-    "./server": "./dist/server/index.jsx",
+    ".": {
+      "default": "./dist/index.js",
+      "types": "./dist/index.d.ts"
+    },
+    "./config": {
+      "default": "./dist/config/index.js",
+      "types": "./dist/config/index.d.ts"
+    },
+    "./server": {
+      "default": "./dist/server/index.jsx",
+      "solid": "./dist/server/index.jsx",
+      "types": "./dist/server/index.d.ts"
+    },
+    "./client": {
+      "default": "./dist/client/index.jsx",
+      "types": "./dist/server/index.d.ts"
```

Comment:
@Brendonovich should this point at ./dist/client/index.d.ts?

Reply:
Definitely lol